style.css

```diff
@@ -88,3 +98,317 @@ button {
   right: 24px;
   border-radius: 8px;
 }
+
+button:hover {
+  color: gray;
+  cursor: pointer;
+}
+
+.second-section {
+  background: #e5e5e5;
+  padding-bottom: 114px;
+  padding-top: 114px;
+  background-size: cover;
+  min-height: calc(100vh - 599px);
+}
+
+.first-phase {
+  display: flex;
+  flex-direction: column;
+  padding: 16px 0 40px 0;
+  box-sizing: border-box;
+  align-items: center;
+  gap: 12px;
+  width: 327px;
+  height: 100%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin-top: 88px;
+  margin-left: 16px;
+}
+
+.phase-one {
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 16px;
+  padding-left: 16px 0 16px 0;
+  align-items: center;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+  overflow-wrap: break-word;
+}
+
+.tonic {
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ OPTIONAL ] To make sure that your code is **DRY**. I would suggest you put this font in the body as a default font because from what I can see it is the only font you are using. This way all your items will have that font by default then can delete it from all other places.

I marked this as optional but I highly advise you to do that because if you do that, you are going to have the same font apply to the **About me** heading in the About section as requested by the previous reviewer, with is not done so far.

#### Do it like this :point_down: 

```
body {
  font-family: 'Poppins', sans-serif;
}
```
#### And remove the font from all other places.